qiskit/compiler/assemble.py

```diff
@@ -36,6 +36,7 @@ def assemble(experiments,
              schedule_los=None, meas_level=MeasLevel.CLASSIFIED,
              meas_return=MeasReturnType.AVERAGE, meas_map=None,
              memory_slot_size=100, rep_time=None, parameter_binds=None,
+             parametric_pulses=None,
```

Comment:
What is the reasoning behind having `parametric_pulses` parsed to assemble? I see below that if it is not given we take the one from the backend.

Reply:
This way the user can override the backend, supported values. This has been the pattern that has been followed so far, but we might have to begin to reconsider soon as these proliferate and append modifications directly to the backend.

Reply:
I would be in favor of that, but for the purposes of this PR we should maintain the same pattern.